tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Capability/Update Capabilities.3678.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Update Capabilities
+    As a Public User
+    I want to update the Capabilities for a Solution
+    So that I can modify who the Solution Capabilities are
+
+Background:
+	Given Capabilities exist
+		| CapabilityName      | CapabilityRef | IsFoundation |
+		| Resource Management | C1            | false        |
+		| Prescribing         | C2            | true         |
+		| Workflow            | C3            | true         |
+	And Suppliers exist
+		| Id    | SupplierName |
+		| Sup 1 | Supplier 1   |
+	And Solutions exist
+		| SolutionID | SolutionName   | SupplierStatusId | SupplierId |
+		| Sln1       | MedicOnline    | 1                | Sup 1      |
+		| Sln2       | TakeTheRedPill | 1                | Sup 1      |
+	And Solutions are linked to Capabilities
+		| Solution       | Capability          |
+		| TakeTheRedPill | Resource Management |
```

Comment:
Has this always used the name of a solution? Could it use the ID instead?

Reply:
Will create a seperate task for this

Reply:
Task: 4894